ui/tsconfig.json

```diff
@@ -11,16 +11,13 @@
     "emitDecoratorMetadata": true,
     "experimentalDecorators": true,
     "target": "es5",
-    "typeRoots": [
-      "node_modules/@types"
-    ],
```

Comment:
Uneccessary. It is included in the default value.
https://www.typescriptlang.org/tsconfig#typeRoots